.github/workflows/check-bioc.yml

```diff
@@ -0,0 +1,291 @@
+## Read more about GitHub actions the features of this GitHub Actions workflow
+## at https://lcolladotor.github.io/biocthis/articles/biocthis.html#use_bioc_github_action
+##
+## For more details, check the biocthis developer notes vignette at
+## https://lcolladotor.github.io/biocthis/articles/biocthis_dev_notes.html
+##
+## You can add this workflow to other packages using:
+## > biocthis::use_bioc_github_action()
+##
+## Using GitHub Actions exposes you to many details about how R packages are
+## compiled and installed in several operating system.s
+### If you need help, please follow the steps listed at
+## https://github.com/r-lib/actions#where-to-find-help
+##
+## If you found an issue specific to biocthis's GHA workflow, please report it
+## with the information that will make it easier for others to help you.
+## Thank you!
+
+## Acronyms:
+## * GHA: GitHub Action
+## * OS: operating system
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+name: R-CMD-check-bioc
+
+## These environment variables control whether to run GHA code later on that is
+## specific to testthat, covr, and pkgdown.
+##
+## If you need to clear the cache of packages, update the number inside
+## cache-version as discussed at https://github.com/r-lib/actions/issues/86.
+## Note that you can always run a GHA test without the cache by using the word
+## "/nocache" in the commit message.
+env:
+  has_testthat: 'true'
+  run_covr: 'true'
+  run_pkgdown: 'false'
+  has_RUnit: 'false'
+  cache-version: 'cache-v1'
+  run_docker: 'false'
+
+jobs:
+  build-check:
+    runs-on: ${{ matrix.config.os }}
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+    container: ${{ matrix.config.cont }}
+    ## Environment variables unique to this job.
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - { os: ubuntu-latest, r: 'devel', bioc: '3.15', cont: "bioconductor/bioconductor_docker:devel", rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest" }
```

Comment:
Does `bioc: '3.15'` indicate the current devel version of bioconductor? Because the bioconductor version numbers change, it might be better to specify it differently (if you can somehow specify "release" or "devel" instead of a version number), or add it as an environment variable at the top (under `run docker`).

It looks like there is a [different but related version](https://github.com/lcolladotor/biocthis/blob/e6e5cb3e122c3112784fcd04536df9ed50021a30/actions/check-bioc.yml) of `check-bioc.yaml` that might do some of this (the `findrversion` step), but I'm not sure if you started there and had to change some of it for some reason.

Reply:
Oh hm, the different version I linked to appears to be from an old commit in `biocthis` and I haven't tracked down this history.

Reply:
I wondered this also, but I used the current version of `biocthis::use_bioc_github_action(biocdocker='devel')` and this is what it gave me. I guess it's just something we'll have to remember to update twice a year.

Reply:
I wondered this also, but I used the current version of `biocthis::use_bioc_github_action(biocdocker='devel')` and this is what it gave me. I guess it's just something we'll have to remember to update twice a year.